ui/redux/reducers/collections.js

```diff
@@ -170,13 +161,17 @@ const collectionsReducer = handleActions(
       });
     },
     [ACTIONS.USER_STATE_POPULATE]: (state, action) => {
-      const { builtinCollections, savedCollections, unpublishedCollections, editedCollections } = action.data;
+      const { builtinCollections, unpublishedCollections, editedCollections } = action.data;
+      const newBuiltin = {
+        [COLS.FAVORITES_ID]: builtinCollections[COLS.FAVORITES_ID],
+        [COLS.WATCH_LATER_ID]: builtinCollections[COLS.WATCH_LATER_ID],
+      };
+
       return {
         ...state,
         edited: editedCollections || state.edited,
         unpublished: unpublishedCollections || state.unpublished,
-        builtin: builtinCollections || state.builtin,
-        saved: savedCollections || state.saved,
+        builtin: newBuiltin || state.unpublished,
       };
     },
```

Comment:
Some of my collections are not appearing in LBRY Desktop.

Is `newBuiltin` meant to erase the old version, and replace with new version that uses `Date.now()` as the timestamp?

- [ ] `builtin: newBuiltin || state.unpublished` -- unpublished is a typo?
  - In Redux, we return the original state reference when we know there is no change. In this case, `newBuiltIn` will always be a new object, so the `||` fallback will never be used.

Anyway, unclear what `newBuiltin` is for, but if it's to override old wallet data, I think we can't do that when LBRY Desktop is still in active use -- they will override each other.  If there is behavioral differences, it might be easier to create new lists and do a one-time action to port over if we detect the odysee version is null and lbry version is non-null.  Then, change the reducer to never touch the existing ones, and GUI will never display them.  

This is one idea to disconnect LBRY and Odysee lists properly, but I still think they should be in sync.
```
favorites
watchlater
favorites_odysee
watchlater_odysee
```